README.md

```diff
@@ -1 +1,33 @@
-#My diary
\ No newline at end of file
+#Author's Haven Front-end
```

Comment:
please make this bold.

Reply:
Good job @cop1fab, I appreciate the task you have done. If you think it is necessary to put the link of the backend repo, it would be great. But I am going to approve it. Thx